audb/core/dependencies.py

```diff
@@ -361,6 +376,37 @@ def version(self, file: str) -> str:
         """
         return self._df.version[file]
 
+    def _add_attachment_file(
```

Comment:
```suggestion
    def _add_attachment(
```

To be inline with `_add_media()` and `_add_meta()`

Reply:
I agree that this at first seems to better fit to `_add_media()` and `_add_meta()`, but there is a difference. As an attachment I would name the object that is stored with an ID under `db.attachments`, but this can have one or several files. To make it more clear that we only add a single file here and not all the file from one attachment, I selected `_add_attachment_file()` as name.